flake.nix

```diff
@@ -2,7 +2,7 @@
   description = "A Nix-based continuous build system";
 
   inputs.nixpkgs.follows = "nix/nixpkgs";
-  inputs.nix.url = "github:NixOS/nix/2.11.0";
+  inputs.nix.url = "github:lheckemann/nix/nixpkgs-22.11";
```

Comment:
Can you point this to 2.12.0?

Reply:
I think it needs to be a branch with https://github.com/NixOS/nix/pull/7367, so `master` unless we backport that to the release branch?

Reply:
(IMO it would be nice to have a branch of hydra that always uses Nix master, along with at least one branch that uses a Nix release.)